main.js

```diff
@@ -0,0 +1,64 @@
+var peoples = [
+	{
+	  title: "Real Estate Agent",
+	  name: "Phil Dunphy",
+	  bio: "Successful real estate agent and father in the show Modern Family. Extremely corny.",
```

Comment:
may want to fix the spacing so the code blocks are more explicit to understand and consistently formatted